src/clib/pio_darray.c

```diff
@@ -171,6 +180,7 @@ int PIOc_write_darray_multi(const int ncid, const int *vid, const int ioid,
         /* Get a buffer. */
         vdesc0->fillbuf = bget(iodesc->holegridsize * vsize * nvars);
 
+	/* ??? */
```

Comment:
copying the fill value into the data buffer for the box rearranger.  This will be overwritten with data where provided.